.github/workflows/python.yml

```diff
@@ -53,16 +55,19 @@ jobs:
         with:
           path: dist/*.tar.gz
 
-  build_linux_wheels:
-
-    name: Build wheels on ubuntu-latest
-    runs-on: ubuntu-20.04
+  # Build wheels
+  build_wheels:
+    name: Build wheels [${{ matrix.os }}]
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         python-version:
           - 3.8
-          # - 3.9
+        os:
+          - ubuntu-latest
```

Comment:
Can we stick to `ubuntu-20.04`? Mostly for reproducibility, even if `cibuildwheel` provide some abstraction better to be conservative.

Reply:
[Done](https://github.com/robotology/idyntree/compare/dfec3d3709a9950899d21286d4afab735070d2f5..b7d12961120b07b0d3cfc9e138f6488ecc081429)